app/views/organisations/_what_we_do.html.erb

```diff
@@ -20,6 +20,10 @@
             </a>
           </p>
       <% end %>
+
+      <% if @organisation.is_dfid? %>
+        <%= render partial: 'uk_aid' %>
```

Comment:
I'm not sure it's worth having a partial for this as the actual code for rendering the UK Aid bit is so small and we're already in a partial separate from the main show view here?

Reply:
Maybe not. I was trying to keep it separate from the rest of the code.